tests/Unit/Numerical/RootFinding/Test_OneDRootFinder.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include <catch.hpp>
+
+#include "Numerical/RootFinding/RootFinder.hpp"
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wparentheses"
+
+namespace {
+double f_free(double x) { return 2.0 - x * x; }
+struct F {
+  double operator()(double x) { return 2.0 - x * x; }
+};
+
+
+TEST_CASE("Unit.Numerical.RootFinding.TOMS748RootSolver", "[Numerical][RootFinding][Unit]") {
+  double abs_tol = 1e-5;
+  double rel_tol = 1e-5;
```

Comment:
These tolerances seem very loose, I assume to make the test faster? I would suggest adding a comment to explain the reason for these values.